framework/api/routes/stock_items/stock_item_router.py

```diff
@@ -1,25 +1,50 @@
 from clapy import IServiceProvider
-from flask import Blueprint, current_app
+from flask import Blueprint, current_app, request
+from varname import nameof
+from application.use_cases.stock_items.create_stock_item.create_stock_item_input_port import CreateStockItemInputPort
+from domain.entities.base_entity import EntityID
+from framework.api.infrastructure.request_object_decorator import request_object
+from framework.api.routes.products.product_router import PRODUCT_ROUTER, get_products_async
+from framework.api.routes.stock_items.create_stock_item_command import CreateStockItemCommand
+from framework.api.routes.stock_items.create_stock_item_presenter import CreateStockItemPresenter
 
 from framework.api.routes.stock_items.get_stock_items_presenter import \
     GetStockItemsPresenter
 from interface_adaptors.controllers.stock_item_controller import \
     StockItemController
 
-STOCK_ITEM_ROUTER = Blueprint("stock_item_router", __name__, url_prefix="/api/stockItems")
+STOCK_ITEM_ROUTER = Blueprint("STOCK_ITEM_ROUTER", __name__, url_prefix="/api/stockItems")
```

Comment:
while working on the stock location i realised our endpoint casing is wrong, should be kebab case for stock-items.

![image](https://github.com/BenTalese/DiscountDora/assets/95552720/aee37df8-89e0-459b-8a1b-92126dce827c)

src: https://api.gov.au/sections/naming-conventions.html